### DISCUSSION TITLE
Can I restart chia GUI?

### DISCUSSION THREAD

**Discussion by Wilkugmo** [swar/Swar-Chia-Plot-Manager#307] [Q&A]
As the title says, I need to restart official chia gui cause I can not connect to the network. But I'm worried if it will kill all the plots I'm making with swar manager.

Hope someone knows what will happen.

**Comment by intdsgn** [comment]
Think of plotting as a completely separate process from the Chia node. In fact, it can be done offline and on separate workstations across various operating systems even. The only link between plotting and the Chia gui is if you happen to specifically use the Chia gui to do your plotting.

Since you are using Svar's Plot Manager to plot via CLI, shutting down or restarting the Chia Blockchain application will have no affect on your plots.

The exception is If you happen to be plotting with Svar and the Chia app at the same time, then those plotting via the Chia app will be void upon closing the app and you will need to delete the corresponding temp files.

Hope that helps!

**Reply by Wilkugmo** [reply]
Thanks for writing this, it did help :)
